pkg/cli/admin/catalog/mirror.go

```diff
@@ -386,6 +391,20 @@ func (o *MirrorCatalogOptions) Run() error {
 	return WriteManifests(o.IOStreams.Out, o.SourceRef, o.DestRef, o.ManifestDir, o.IcspScope, mapping)
 }
 
+func (o *MirrorCatalogOptions) PostRun() error {
+	// If we have NOT set --path, the TempDir is set to true and we will delete
+	// Temporary folder.
+	if o.TempDir {
+		dir := strings.Split(o.DatabasePath, ":")
+		if err := os.RemoveAll(dir[1]); err != nil {
```

Comment:
Do we need to verify that this index exists by checking the length of the `dir` split string?

Reply:
Right, it should but I guess nothing wrong with adding lenght check so we dont end up in  index out of bound error.

Reply:
@zerodayz It looks like this PR needs a rebase now - I think these changes still look reasonable to me. Small nit/nothing blocking: might be worth creating a variable that's initialized from the `dir[1]` as we use that in multiple places here.